wiki/fr/fr-Client-Troubleshooting.md

```diff
@@ -79,9 +79,10 @@ Voir [cette discussion du forum](https://sourceforge.net/p/llcon/discussion/5335
 ### Vous utilisez un Mac et votre entrée n'est pas entendue ?
 
 (Avec les remerciements de [Mark Anthony De Souza](https://www.facebook.com/groups/619274602254947/permalink/765122847670121/?comment_id=765525034296569))
-Vous n'avez peut-être pas répondu "Oui" à la question "Jamulus veut accéder à votre microphone".  Pour résoudre ce problème :
+
+Vous n'avez peut-être pas répondu "Oui" à la question "Jamulus veut accéder à votre microphone". Pour résoudre ce problème :
 * Allez dans `Préférences` > `Sécurité et confidentialité` > onglet `Privacy`.
-* Trouvez "Microphone" sur la gauche et assurez-vous que "Jamulus" est activé sur la liste de droite.
+* Trouvez `Microphone` sur la gauche et assurez-vous que `Jamulus` est activé sur la liste de droite.
```

Comment:
Why was this changed? \` is used for code/CLI commands usually.

Reply:
You'd have to ask @jujudusud. I didn't change any of the FR content except obvious missing **, _  and links

Reply:
Ok. Thanks for this informaiton. Seems as if I need to invest some more time in the website before we can release it.

Reply:
Actually, it's like that in the original EN document - @jujudusud simply followed the same style.

Reply:
Yes, same as EN for me. only translating ... ;-)